sanitize_test.go

```diff
@@ -50,6 +74,7 @@ func Test_Sanitize_Options(t *testing.T) {
 		Name            string `abcde:"max=5,trim,lower"`
 		Birthday        string `abcde:"date"`
 		PersonalWebsite string `abcde:"xss,trim"`
+		Breed           string `abcde:"capfirst"`
```

Comment:
Can you add a test case that would show the expected result if the capfirst custom sanitizer function, as defined here, were used on a non-string field? For example, am `int` field?

Reply:
Done